src/cascadia/UnitTests_TerminalCore/SelectionTest.cpp

```diff
@@ -709,7 +709,7 @@ namespace TerminalCoreUnitTests
                 ValidateLinearSelection(term, { 5, 10 }, { 10, 10 });

```

Comment:
I don't understand why these coordinates did not become exclusive

Reply:
Good catch! Looked into it more. Since we're moving behind the pivot, end is actually inclusive! (which, admittedly, is pretty confusing)

Leaving this thread unresolved in case we want to make any changes because of this